### PR TITLE
Modify the inaccuracies in the TLS description

### PR DESCRIPTION
**Pull request by lianan-352889950**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Some descriptions were inaccurate and did not align with the current state of TiDB Cloud products. These inaccuracies caused inconvenience to users. To address this issue, adjustments have been made to ensure that the document aligns as closely as possible with the current status of TiDB Cloud products.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md [hunk 1]
```diff
@@ -28,16 +28,17 @@ In the [TiDB Cloud console](https://tidbcloud.com/), you can get examples of dif
...
+5. Under **Step 2: Download CA Cert**, click **Download CA Cert** to download it locally for client TLS configuration. The CA Cert ensures that the TLS connection is secure and reliable.
 
     > **Note:**
     >
-    > After downloading your TiDB Dedicated cluster CA, you can store it in the default storage path of your operating system, or specify another storage path. You need to replace the CA path in the code example with your own cluster CA path in the subsequent steps.
+    > After downloading your CA Cert, you can store it in the default storage path of your operating system, or specify another storage path. You need to replace the CA Cert path in the code example with your own CA Cert path in the subsequent steps.  
+   > TiDB Dedicated does not force clients to use TLS connections, and user-defined configuration of the `require_secure_transport` variable is not supported on [TiDB Dedicated](https://docs.pingcap.com/tidbcloud/system-variables#require_secure_transport-new-in-v610) currently.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md:41]:
```suggestion
    > - After downloading your CA Cert, you can store it in the default storage path of your operating system, or specify another storage path. You need to replace the CA Cert path in the code example with your own CA Cert path in the subsequent steps.  
   > - TiDB Dedicated does not force clients to use TLS connections, and user-defined configuration of the  [`require_secure_transport`](/system-variables.md#require_secure_transport-new-in-v610) variable is currently not supported on [TiDB Dedicated](https://docs.pingcap.com/tidbcloud/system-variables#require_secure_transport-new-in-v610).
```

File: tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md [hunk 2]
```diff
@@ -298,4 +299,4 @@ For security reasons, TiDB Dedicated only supports TLS 1.2 and TLS 1.3, and does
 
 No.
 
-TiDB Dedicated only supports one-way TLS authentication, and does not support two-way TLS authentication currently. If you need two-way TLS authentication, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).
+TiDB Dedicated only supports one-way TLS authentication, and does not support two-way TLS authentication currently. Please do not make custom configurations of `ssl_cert` and `ssl_key` parameters on TiDB Dedicated. If you need two-way TLS authentication, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md:302]:
```suggestion
TiDB Dedicated only supports one-way TLS authentication, and does not support two-way TLS authentication currently. If you need two-way TLS authentication, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).
```

Comment (qiancai) [reply tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md:302]: reason: these two variables are read-only on TiDB Cloud.

File: tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md [hunk 3]
```diff
@@ -28,16 +28,17 @@ In the [TiDB Cloud console](https://tidbcloud.com/), you can get examples of dif
...
 3. On the **Standard Connection** tab of this dialog, follow the three steps to set up the TLS connection.
    - Step 1：Create traffic filter
-   - Step 2：Download TiDB cluster CA
+   - Step 2：Download CA Cert
    - Step 3：Connect with an SQL client
 
 4. Under **Step 1: Create traffic filter** in the dialog, configure the IP addresses that are allowed to access your cluster. For more information, see [Configure an IP access list in standard connection](/tidb-cloud/configure-ip-access-list.md#configure-an-ip-access-list-in-standard-connection).
 
-5. Under **Step 2: Download TiDB cluster CA**, click **Download TiDB cluster CA** to download it locally for client TLS configuration. The TiDB cluster CA ensures that the TLS connection is secure and reliable.
+5. Under **Step 2: Download CA Cert**, click **Download CA Cert** to download it locally for client TLS configuration. The CA Cert ensures that the TLS connection is secure and reliable.
```
Comment (Yuiham) [inline tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md:36]:
```suggestion
5. Under **Step 2: Download CA Cert**, click **Download CA Cert** to download it locally for client TLS configuration. The CA certificate ensures that the TLS connection is secure and reliable.
```

Comment (lianan-352889950) [reply tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md:36]: CA certificate and CA Cert both mean the same thing. Our competing product is also described as CA Cert. There is no need to modify it here.

File: tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md [hunk 4]
```diff
@@ -28,16 +28,17 @@ In the [TiDB Cloud console](https://tidbcloud.com/), you can get examples of dif
...
 
     > **Note:**
     >
-    > After downloading your TiDB Dedicated cluster CA, you can store it in the default storage path of your operating system, or specify another storage path. You need to replace the CA path in the code example with your own cluster CA path in the subsequent steps.
+    > - After downloading your CA Cert, you can store it in the default storage path of your operating system, or specify another storage path. You need to replace the CA Cert path in the code example with your own CA Cert path in the subsequent steps.  
+   > - TiDB Dedicated does not force clients to use TLS connections, and user-defined configuration of the [`require_secure_transport`](/system-variables.md#require_secure_transport-new-in-v610) variable is currently not supported on [TiDB Dedicated](https://docs.pingcap.com/tidbcloud/system-variables#require_secure_transport-new-in-v610).
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md:41]:
```suggestion
   > - TiDB Dedicated does not force clients to use TLS connections, and user-defined configuration of the [`require_secure_transport`](/system-variables.md#require_secure_transport-new-in-v610) variable is currently not supported on TiDB Dedicated.
```

**Review by qiancai — APPROVED** [review]

File: tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md [hunk 5]
```diff
@@ -28,16 +28,17 @@ In the [TiDB Cloud console](https://tidbcloud.com/), you can get examples of dif
...
 
-5. Under **Step 2: Download TiDB cluster CA**, click **Download TiDB cluster CA** to download it locally for client TLS configuration. The TiDB cluster CA ensures that the TLS connection is secure and reliable.
+5. Under **Step 2: Download CA Cert**, click **Download CA Cert** to download it locally for client TLS configuration. The CA Cert ensures that the TLS connection is secure and reliable.
 
     > **Note:**
     >
-    > After downloading your TiDB Dedicated cluster CA, you can store it in the default storage path of your operating system, or specify another storage path. You need to replace the CA path in the code example with your own cluster CA path in the subsequent steps.
+    > - After downloading your CA Cert, you can store it in the default storage path of your operating system, or specify another storage path. You need to replace the CA Cert path in the code example with your own CA Cert path in the subsequent steps.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md:40]:
```suggestion
    > - You can store the downloaded CA Cert in the default storage path of your operating system, or specify another storage path. You need to replace the CA Cert path in the code example with your own CA Cert path in the subsequent steps.  
```

**Review by hfxsd — APPROVED** [review]

File: tidb-cloud/connect-via-standard-connection.md [hunk 6]
```diff
@@ -38,11 +38,11 @@ To connect to a TiDB Dedicated cluster via standard connection, take the followi
 
     3. Click **Create Filter** to confirm the changes.
 
-4. Under **Step 2: Download TiDB cluster CA** in the dialog, click **Download TiDB cluster CA** for TLS connection to TiDB clusters. The TiDB cluster CA supports TLS 1.2 version by default.
+4. Under **Step 2: Download CA Cert** in the dialog, click **Download CA Cert** for TLS connection to TiDB clusters. The CA Cert supports TLS 1.2 version by default.
```
Comment (qiancai) [inline tidb-cloud/connect-via-standard-connection.md:41]:
```suggestion
4. Under **Step 2: Download CA cert** in the dialog, click **Download CA cert** for TLS connection to TiDB clusters. The CA cert supports TLS 1.2 version by default.
```

File: tidb-cloud/connect-via-standard-connection.md [hunk 7]
```diff
@@ -38,11 +38,11 @@ To connect to a TiDB Dedicated cluster via standard connection, take the followi
...
 
     > **Note:**
     >
-    > - The TiDB cluster CA is only available for TiDB Dedicated clusters.
+    > - The CA Cert is only available for TiDB Dedicated clusters.
```
Comment (qiancai) [inline tidb-cloud/connect-via-standard-connection.md:45]:
```suggestion
    > - The CA cert is only available for TiDB Dedicated clusters.
```

File: tidb-cloud/migrate-from-mysql-using-aws-dms.md [hunk 8]
```diff
@@ -100,7 +100,7 @@ AWS DMS does not support replicating `DROP TABLE`.
 
 3. Under **Step 1: Create traffic filter** in the dialog, click **Edit**, enter the public and private network IP addresses that you copied from the AWS DMS console, and then click **Update Filter**. It is recommended to add the public IP address and private IP address of the AWS DMS replication instance to the TiDB cluster traffic filter at the same time. Otherwise, AWS DMS might not be able to connect to the TiDB cluster in some scenarios.
 
-4. Click **Download TiDB cluster CA** to download the CA certificate. Under **Step 3: Connect with a SQL client** in the dialog, take a note of the `-u`, `-h`, and `-P` information in the connection string for later use.
+4. Click **Download CA Cert** to download the CA certificate. Under **Step 3: Connect with a SQL client** in the dialog, take a note of the `-u`, `-h`, and `-P` information in the connection string for later use.
```
Comment (qiancai) [inline tidb-cloud/migrate-from-mysql-using-aws-dms.md:103]:
```suggestion
4. Click **Download CA cert** to download the CA certificate. Under **Step 3: Connect with a SQL client** in the dialog, take a note of the `-u`, `-h`, and `-P` information in the connection string for later use.
```

File: tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md [hunk 9]
```diff
@@ -28,16 +28,17 @@ In the [TiDB Cloud console](https://tidbcloud.com/), you can get examples of dif
 
 3. On the **Standard Connection** tab of this dialog, follow the three steps to set up the TLS connection.
    - Step 1：Create traffic filter
-   - Step 2：Download TiDB cluster CA
+   - Step 2：Download CA Cert
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md:31]:
```suggestion
   - Step 2：Download CA cert
```

File: tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md (hunk 3 above)
Comment (qiancai) [inline tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md:36]:
```suggestion
5. Under **Step 2: Download CA cert**, click **Download CA cert** to download it locally for client TLS configuration. The CA cert ensures that the TLS connection is secure and reliable.
```

File: tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md [hunk 10]
```diff
@@ -28,16 +28,17 @@ In the [TiDB Cloud console](https://tidbcloud.com/), you can get examples of dif
...
 
-5. Under **Step 2: Download TiDB cluster CA**, click **Download TiDB cluster CA** to download it locally for client TLS configuration. The TiDB cluster CA ensures that the TLS connection is secure and reliable.
+5. Under **Step 2: Download CA Cert**, click **Download CA Cert** to download it locally for client TLS configuration. The CA Cert ensures that the TLS connection is secure and reliable.
 
     > **Note:**
     >
-    > After downloading your TiDB Dedicated cluster CA, you can store it in the default storage path of your operating system, or specify another storage path. You need to replace the CA path in the code example with your own cluster CA path in the subsequent steps.
+    > - You can store the downloaded CA Cert in the default storage path of your operating system, or specify another storage path. You need to replace the CA Cert path in the code example with your own CA Cert path in the subsequent steps.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md:40]:
```suggestion
    > - You can store the downloaded CA cert in the default storage path of your operating system, or specify another storage path. You need to replace the CA cert path in the code example with your own CA cert path in the subsequent steps.  
```

**Review by Yuiham — APPROVED** [review]

**Pull request merged**